resources/public/api/conf/1.0/lossClaims_amend.raml

```diff
@@ -0,0 +1,25 @@
+is:
+  - acceptJson
+  - contentTypeJson
+  - testScenarioHeader
+  - formatNino
```

Comment:
Scenario text should read "The format of the supplied NINO field is not valid" (across all endpoints).